stdlib/public/TensorFlow/Utilities.swift

```diff
@@ -63,6 +63,9 @@ func checkOk(_ s: CTFStatus?, file: StaticString = #file, line: UInt = #line) {
 // Type aliases
 //===----------------------------------------------------------------------===//
 
+// For all pointer related types below, before assigning a C pointer to one such
+// object, caller should check to make sure the pointer is not NULL.
```

Comment:
Nit: rephrase to sth like
`Before assigning an OpaquePointer to one of the pointer type aliases below, caller should check that the pointer is not NULL.`

Reply:
Note an OpaquePointer cannot be nil though (OpaquePointer? can).

Here the comment is intentionally worded for a C audience (not Swift), since these pointer types are C data types.

Reply:
I see.

How about:
`Before assigning a C pointer to one of the pointer type aliases below, caller should check that the pointer is not NULL.`

Reply:
Done.